apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/logging/core/Converter.java

```diff
@@ -19,10 +19,13 @@
 package org.apache.skywalking.apm.agent.core.logging.core;
 
 /**
- * The Converter, It is used to convert the LogEvent to the String.
+ * The Converter, it is used to convert the LogEvent to the String.
  */
 public interface Converter {
 
     String convert(LogEvent logEvent);
 
+    default String getKey() {
```

Comment:
Once you add this, why need a default implementation?

Reply:
> Once you add this, why need a default implementation?

We don't need to implement `getKey()` for `LiteralConverter`